.github/workflows/main.yaml

```diff
@@ -3,7 +3,9 @@ on:
   push:
     branches:
       - main
-      
+  schedule:
```

Comment:
Something occurred to me recently - please could you add a bit to the readme that explains that we run this nightly and why?

Reply:
Done

Reply:
Just second what LB mentioned as I'm not seeing a ReadMe

Reply:
Not sure why it wasn't showing up, I've also added the same schedule comment to validate.yaml

Reply:
Looks great!